AssertionError: Number must be an unsigned integer, not a float

as discussed in https://github.com/adafruit/Adafruit_CircuitPython_RSA/pull/10, the simpletest for this library fails:

on 5.0
```
Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.
code.py output:
Generating keypair...
Encrypting message...
Traceback (most recent call last):
  File "code.py", line 16, in <module>
  File "/lib/adafruit_rsa/pkcs1.py", line 173, in encrypt
  File "/lib/adafruit_rsa/transform.py", line 181, in int2bytes
AssertionError: Number must be an unsigned integer, not a float.
```

and on 4.1.2
```
Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.
code.py output:
Generating keypair...
Encrypting message...
Traceback (most recent call last):
  File "code.py", line 16, in <module>
  File "/lib/adafruit_rsa/pkcs1.py", line 173, in encrypt
  File "/lib/adafruit_rsa/transform.py", line 177, in int2bytes
AssertionError: Number must be an unsigned integer, not a float.```
```

--- a/adafruit_rsa/transform.py
+++ b/adafruit_rsa/transform.py
@@ -178,7 +178,7 @@ def int2bytes(number, fill_size=None, chunk_size=None, overflow=False):
         raise ValueError("You can either fill or pad chunks, but not both")
 
     # Ensure these are integers.
-    assert number % 1 == 0, "Number must be an unsigned integer, not a float."
+    assert isinstance(number, int), "Number must be an unsigned integer, not a float."
 
     raw_bytes = b""
 

--- a/adafruit_rsa/transform.py
+++ b/adafruit_rsa/transform.py
@@ -178,7 +178,7 @@ def int2bytes(number, fill_size=None, chunk_size=None, overflow=False):
         raise ValueError("You can either fill or pad chunks, but not both")
 
     # Ensure these are integers.
-    assert number & 1 == 0, "Number must be an unsigned integer, not a float."
+    assert number % 1 == 0, "Number must be an unsigned integer, not a float."
 
     raw_bytes = b""
 

